humble/distribution.yaml

```diff
@@ -2978,6 +2978,16 @@ repositories:
       type: git
       url: https://github.com/lgsvl/lgsvl_msgs.git
       version: foxy-devel
+  libcaer:
```

Comment:
Can you please explain the relationship between `libcaer` here and `libcaer_driver` below?

Reply:
libcaer_driver depends on libcaer (a library that provides access to all of Inilab's devices).
I first tried to compile and install libcaer directly inside the libcaer_driver workspace but couldn't get it to work so I decided to provide it as a separate package. That will also make it easier if somebody wants to access libcaer directly for other ROS projects.